h2o-hadoop/scripts/docker/common/hdfs_create_h2o_home

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+
```

Comment:
How do you make sure the HDFS service is fully initialized before running this command?